docs/sources/operators-guide/configure/configuring-out-of-order-samples-ingestion.md

```diff
@@ -64,3 +52,19 @@ Similar to the problem above with query caching, the samples recorded via the re
 So you should expect some difference in results if you happen to run the raw query of the recording rule. The difference highly depends on your out-of-order ingestion pattern.
 
 If you happen to have a shorter `out_of_order_time_window`, say less than 10 minutes, then you can use `-ruler.evaluation-delay-duration` to delay your rule evaluation up to that time.
+
+## Meaning of out-of-order
+
+Grafana Mimir and the Prometheus TSDB understand out-of-order as follows.
```

Comment:
How about instead of "understand out-of-order as follows" we say something like "previously, Mimir and Prometheus TSDB had a couple of rules over what timestamps are accepted" ?